code/game/gamemodes/traitor/traitor.dm

```diff
@@ -44,6 +52,8 @@
 	else
 		num_traitors = max(1, min(num_players(), traitors_possible))
 
+	var/num_contractors = max(min_contractors, CEILING(num_traitors * contractor_traitor_ratio, 1))
```

Comment:
I feel like this would make contractors a rarer occurrence than they should be, with how RNG is sometimes. The current method ensures there's at least always one potential contractor, scaling up.

Reply:
This code also ensures there's at least `min_cotractors` potential contractors, while allowing for some RNG swings (which, I argue, are a good thing):
```dm
var/num_contractors = 0
for(var/i =0, i < num_traitors, i++)
  if(prob(contractor_traitor_ratio * 100))
    num_contractors += 1
if(num_contractors < min_contractors)
  num_contractors = min_contractors
```